pom.xml

```diff
@@ -56,87 +49,14 @@
 			<artifactId>slf4j-api</artifactId>
 			<version>1.6.3</version>
 		</dependency>
-		<dependency>
-			<groupId>javax</groupId>
-			<artifactId>javaee-web-api</artifactId>
-			<version>6.0</version>
-			<scope>provided</scope>
-		</dependency>
-		<dependency>
-			<groupId>com.lowagie</groupId>
-			<artifactId>itext</artifactId>
```

Comment:
@evernat Removed `javaee-web-api` and `itext` dependencies since they don't seem to be used anywhere. Could you please review this? Is there a reason to not remove them?